README.md

```diff
@@ -0,0 +1,32 @@
+# tskit-dev/administrative <img align="right" width="145" height="90" src="tskit_logo.ink.svg">
+This repository contains project management documents and discussions for
+development of the tree sequence toolkit (tskit). Tree sequences are a flexible and
+compact way of representing genetic history and storing DNA variation data.
+
+## Contributing
+
+We encourage contributions to improve any of the
+[tskit development repositories](https://github.com/tskit-dev).
+Improvements to documentation are particularly welcome, and are likely to be accepted
+quickly. 
+
+The preferred way to contribute is to create your own fork of the appropriate
+repository (for instance, `tskit-dev/tskit`), make changes, and submit your changes as a
+pull request (PR). A simple way to do this is to create a topic branch based on upstream,
```

Comment:
We need to tell people how to set up an upstream remote. Perhaps point to docs here: https://help.github.com/articles/configuring-a-remote-for-a-fork/?

Reply:
When I do this on GitHub (by "opening in desktop", it seems to set the remote automatically. I'll phrase it appropriately